### ISSUE TITLE
Uninitialized string offset 0

### ISSUE THREAD

**Issue by nitech** [verbb/postie#120]
### Describe the bug

Going to Plugin Settings, I get this error: 

![image](https://github.com/verbb/postie/assets/1279436/2261a647-47e5-469e-bd2f-a9ba3d33c3e6)

URL is https://amatec.ddev.site/aure/postie/settings?site=amatecNorsk

### Steps to reproduce

Craft Commerce 4.5.1.1

### Craft CMS version

4.8.1

### Plugin version

3.1.5

### Multi-site?

Yes

**Comment by nitech** [comment]
I removed all providers inside postie.php excep `Bring` and then it loaded. Seems there's something wrong with the configuration of one of the providers. Here's the modified code:

```
    private function _getProviderTypes(): array
    {
        $providerTypes = [
            Bring::class,
        ];

        $event = new RegisterProviderTypesEvent([
            'providerTypes' => $providerTypes,
        ]);

        $this->trigger(self::EVENT_REGISTER_PROVIDER_TYPES, $event);

        return $event->providerTypes;
    }

    private function _createProvider($providerType): ProviderInterface
    {
        return new $providerType;
    } 
```

**Comment by nitech** [comment]
Seems it's the `NewZealandPost::class,`

**Comment by engram-design** [comment]
Are you able to send your `config/project/project.yaml` file through to `support@verbb.io`? I would need to replicate your project config (the saved settings for providers) for this one I believe.

**Comment by hassanmah** [comment]
@engram-design I'm also getting this after updating to Commerce 4.5.2 and Craft 4.8.1 

I'm using `"verbb/postie": "dev-craft-4 as 3.1.4.",` as per https://github.com/verbb/postie/issues/119#issuecomment-1925553333

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/postie:"dev-craft-4 as 3.1.5"`.

**Comment by engram-design** [comment]
Fixed in [3.1.6](https://github.com/verbb/postie/releases/tag/3.1.6)
